src/main.py

```diff
@@ -5,8 +5,10 @@
 from task3 import ClassFuncDecorator, rankings
 from task4 import ClassFuncDecorator2
 
+test_decorator = decorator1
```

Comment:
It's easier to have such a variable to avoid writing the tested decorator before the definition of every function